backend/src/main/java/team31/codesuqad/sidedish/controller/CategoryController.java

```diff
@@ -1,22 +1,24 @@
 package team31.codesuqad.sidedish.controller;
 
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
-import team31.codesuqad.sidedish.controller.dto.CategoryNames;
+import team31.codesuqad.sidedish.controller.dto.AllCategoriesResponse;
 import team31.codesuqad.sidedish.service.CategoryService;
 
 @RestController
+@CrossOrigin(origins = "*", allowedHeaders = "*")
 public class CategoryController {
 
-    private CategoryService categoryService;
+    private final CategoryService categoryService;
 
     public CategoryController(CategoryService categoryService) {
         this.categoryService = categoryService;
     }
 
-    @GetMapping("/api/dish/categorynames")
-    public CategoryNames getCategoryNames() {
-        return categoryService.findAll();
+    @GetMapping("/api/dish/categories")
+    public AllCategoriesResponse getCategories() {
+        return categoryService.findCategories();
```

Comment:
메소드, dto이름을 잘 수정해 주셨네요 👍

Reply:
url의 dish는 없어도 충분할 것 같습니다 ;)
```suggestion
    @GetMapping("/api/categories")
    public AllCategoriesResponse getCategories() {
        return categoryService.findCategories();
```